### PR TITLE
character swap filter. 

### PR DESCRIPTION
**Pull request by FreddieHorn**
Returns a list.

### CODE REVIEW

**Comment by FreddieHorn** [comment]
Its a test version for the interface

File: src/filters.py [hunk 1]
```diff
@@ -1,9 +1,42 @@
+from source_dicts import SourceDictionaries
+from itertools import product
 """
 Module containing all micro filers for word validation
 """
 
+def charswap_filter(word: str):
```
Comment (SunBear1) [inline src/filters.py:7]:
```suggestion
def charswap_filter(word: str) -> list:
```

File: src/filters.py [hunk 2]
```diff
@@ -1,9 +1,42 @@
...
+    #         return False
+    return possible_profanities
+
 
 def example_filer(word: str):
     if word == "kurva":
         return False
     return True
+
+if __name__ == "__main__":
```
Comment (SunBear1) [inline src/filters.py:40]: Remove this main please 💯

File: src/source_dicts.py [hunk 3]
```diff
@@ -7,3 +7,17 @@
...
+    raw_swaps: dict = yml_swap_data["swaps"]
+    modified_swaps = {}
+    for key, values in raw_swaps.items():
+        for value in values:
+            modified_swaps[value] = key
+    sorted_swaps = dict(sorted(modified_swaps.items(), key=lambda item: len(item[0]), reverse=True))
+    POSSIBLE_SWAPS: dict = sorted_swaps
+    
+    
+
```
Comment (SunBear1) [inline src/source_dicts.py:23]: Add an empty line at the end of this file

**Review by SunBear1 — CHANGES_REQUESTED** [review]

**Review by jereml99 — APPROVED** [review]

**Review by SunBear1 — APPROVED** [review]
Looking good, didn't want to delay merging by some picky comments. Happy merging 🏆

**Review by jereml99 — APPROVED** [review]

**Pull request merged**